CONTRIBUTING.md

```diff
@@ -59,3 +86,8 @@ If your code intentionally changes some of the existing snapshots, simply blow a
 rm -rf test/output
 yarn test
 ```
+
+## Submitting a pull request
+Once you have completed your feature and confirmed that all test have passed using `yarn test`, you should push your changes up to GitHub and submit a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
```

Comment:
We mention pull requests up above (I suggest we move this GitHub link above), so maybe this section could instead be more specific about our expectations for what makes a good pull request, or a good feature? Like, please include a motivating example, or why you think this feature will be useful. Or, please consider filing an issue, or checking for existing issues, before submitting a pull request, in case someone else has already thought about this topic. Or perhaps to start, this section could instead be called `## Documentation`, and delete this sentence?

Reply:
I switched the last section to **Documentation** and moved the Observable instructions there.